### ISSUE TITLE
Dymo is printing three blank labels for each label it prints

### ISSUE THREAD

**Issue by amcewen** [DoESLiverpool/somebody-should#1284]
I used the Dymo label printer on the Vinyl Cutter PC earlier to print two labels.  Each time it also spooled through an additional 3 blank labels, and after the first label I printed the printer stopped responding and I had to power-cycle it before I could print the second label.

I was printing on the white labels, in the left-hand side of the printer.

I haven't had any problems like that with it in the past.

**Comment by MatthewCroughan** [comment]
I forgot to make an issue about this yesterday as I was heading home, I'm catching up on the issues now and am going to make one relating to reorganizing that area in a moment.

This has been happening since yesterday, when I ejected a label. I ejected the label and I believe a bit of it ripped off inside the mechanism as it splayed from its reel. My thoughts are that it isn't sensing one of the black bars correctly due to this. I think it may be fixable with a scalpel, but didn't want to try and do that until I got thoughts on it.

**Comment by MatthewCroughan** [comment]
@amcewen is this no longer an issue?

**Comment by zarino** [comment]
I’ve got both sides of the Dymo printer working. The left-hand side wasn’t playing ball initially (PC kept complaining that there was no paper on that side) but through a combination of feeding with the blue button, and ejecting with the ejector button, I got it printing.

At first, it was printing labels horizontally offset:

![IMG_1514](https://user-images.githubusercontent.com/739624/70375960-fb49d800-18fb-11ea-9bef-c3cf6f6ef55f.jpg)

Which is a bit weird. I used the eject button to feed it through a bit more, bit by bit, until it looked like the label was in the right position, and now it’s all fine:

![IMG_1515](https://user-images.githubusercontent.com/739624/70375962-03097c80-18fc-11ea-96b8-95a4cc62f44a.jpg)

And I can confirm it’s *not* printing three labels at a time. Just one! So I’m going to close this ticket.
